dynamic_models/schema.py

```diff
@@ -1,11 +1,12 @@
 """Wrapper functions for performing runtime schema changes."""
-from django.db import connection
+from django.db import connections
 from django.db.utils import ProgrammingError
 
 
 class ModelSchemaEditor:
-    def __init__(self, initial_model=None):
+    def __init__(self, initial_model=None, db_name='default'):
```

Comment:
Django defines this default DB alias here at `django.db.utils.DEFAULT_DB_ALIAS`. It would be nice to use that constant here